src/main/resources/org/jenkinsci/plugins/workflow/steps/scm/SCMStep/help.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    This is a special step that will checkout using the configuration options offered by any Pipeline-compatible SCM plugin.
```

Comment:
```suggestion
    This is a special step that checks out the code using the configuration options offered by any Pipeline-compatible SCM plugin.
```
We prefer present tense to future in a case like this.
The proper verb for is "check out" rather than "checkout" and it needs an object

Reply:
Alright, @StackScribe that's well noted.
Thanks for the correction.

Reply:
@Onyimatics you can use the "Commit suggestion" to apply changes that others have suggested.

![image](https://user-images.githubusercontent.com/156685/113712286-2cc6e500-96a3-11eb-92b2-ba910fe4626c.png)

Reply:
@MarkEWaite Thanks, well noted.